docs/data_management/data_responsibilities.md

```diff
@@ -0,0 +1,78 @@
+# Data Access Authorization and Responsibility
```

Comment:
Let's change the title to "Research Data Responsibilities". Change the page name as well.